x/oracle/types/asset.go

```diff
@@ -7,17 +7,17 @@ import (
 )
 
 const (
-	UmeeDenom       string = appparams.BondDenom
-	UmeeSymbol      string = "umee"
-	UmeeExponent           = uint32(6)
-	USDDenom        string = "USD"
-	BlocksPerMinute        = uint64(10)
-	BlocksPerHour          = BlocksPerMinute * 60
-	BlocksPerDay           = BlocksPerHour * 24
-	BlocksPerWeek          = BlocksPerDay * 7
-	BlocksPerMonth         = BlocksPerDay * 30
-	BlocksPerYear          = BlocksPerDay * 365
-	MicroUnit              = int64(1e6)
+	UmeeDenom       = appparams.BondDenom
+	UmeeSymbol      = "UMEE"
```

Comment:
I changed lowercase to UpperCase because we are storing all denoms to Uppercase in our leverage and oracle acceptedlist

Reply:
Note: [mainnet oracle accept list](https://umee-api.polkachu.com/umee/oracle/v1/params) has some lowercase so be careful with anything that uses those constants

e.g. `"symbol_denom": "umee"` and for `"atom"`, `"stATOM"`, `"stOSMO"` as well.

Mainnet [leverage](https://umee-api.polkachu.com/umee/leverage/v1/registered_tokens) only has (partial) lowercase for `"stATOM"`